Docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+services:
+  frontend:
```

Comment:
I think it would be nice to have [Healthcheck](https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck) on our frontend and backend services. Healthcheck is an action to check if our service is still running and if it is still responding. For example, our backend app could have an endpoint to return the current version and the healthcheck will be sending requests on that endpoint and that will ensure us that our app is working correctly.

Reply:
While I agree that health checks can be useful I believe that We don't need to add them now. 
In my opinion, frontend does not need a health check at all as the Docker image is used only for development/debugging purposes as it will most likely be deployed to GitHub pages. As for the backend, it might be useful however again the purpose of this docker-compose is for development/debugging and not deployment.  We would need to use some service for deployment which monitors this (like Azure Web Apps) which will be much more expensive than a cheap VPS.

So to sum it up I would say We might want to add a health check later on for the backend. But I would skip this, for now, to start working on an actual app code. 

What do You think?

Reply:
Ok, I see Your point. I agree, this could be added anytime. Let's focus on the app code.